### PR TITLE
CLOUDP-127785: MongoCLI replaced with Atlas API

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### All Submissions:

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).
* [ ] Update docs/release-notes/release-notes.md if your changes should be included in the release notes for the next release.

### CODE REVIEW

**Review by fabritsius — APPROVED** [review]
Looks Great 👍

**Review by Sugar-pack — APPROVED** [review]
LGTM

File: test/e2e/actions/helm_related.go [hunk 1]
```diff
@@ -29,10 +32,20 @@ func HelmDefaultUpgradeResouces(data *model.TestDataProvider) {
 			WaitDeployment(data.Resources, strconv.Itoa(generation+1))
 			ExpectWithOffset(1, data.Resources.ProjectID).ShouldNot(BeEmpty())
 		})
-		updatedDeployment := mongocli.GetDeploymentsInfo(data.Resources.ProjectID, data.Resources.Deployments[0].Spec.GetDeploymentName())
+		aClient := atlas.NewClientOrFail()
```
Comment (fabritsius) [inline test/e2e/actions/helm_related.go:35]: We creating new client all each time, we can try to reuse the same one. Not critical thought and can be left as is :)

**Review by fabritsius — APPROVED** [review]
Looks Good 👌

File: test/e2e/actions/helm_related.go (hunk 1 above)
Comment (igor-karpukhin) [reply test/e2e/actions/helm_related.go]: Fixed!

**Pull request merged**